app/_includes/md/kic/deploy-kic.md

```diff
@@ -1,43 +1,49 @@
 ## Deploy the {{site.kic_product_name}} {#deploy-kic}
 
-Deploy the {{site.kic_product_name}} using `kubectl`:
+You can deploy the {{site.kic_product_name}} on minikube using `kubectl` or `Helm Charts`:
```

Comment:
The deployment target can be any k8s cluster, not just minikube. This include is used in multiple pages:

```
❯ fgrep -lr "deploy-kic.md" app/_src/kubernetes-ingress-controller
app/_src/kubernetes-ingress-controller/guides/cert-manager.md
app/_src/kubernetes-ingress-controller/deployment/gke.md
app/_src/kubernetes-ingress-controller/deployment/aks.md
app/_src/kubernetes-ingress-controller/deployment/eks.md
app/_src/kubernetes-ingress-controller/deployment/minikube.md
```

Reply:
agreed!!!